apps/frontend/core-dapp/src/features/ppo/stake/StakeUnstakeLayout.tsx

```diff
@@ -56,11 +57,10 @@ const StakeUnstakeLayout: React.FC<{
   const content = pageMap[tab]
 
   const buttonDisabled =
-    disableMocks ||
-    loading ||
-    !(isStake ? isCurrentStakingValueValid : isCurrentUnstakingValueValid)
+    staking || loading || !(isStake ? isCurrentStakingValueValid : isCurrentUnstakingValueValid)
```

Comment:
Then staking here can be removed since loading will be true when staking is true
```suggestion
    loading || !(isStake ? isCurrentStakingValueValid : isCurrentUnstakingValueValid)
```